azure-pipelines.yml

```diff
@@ -59,23 +59,19 @@ stages:
 
           - bash: |
               conda config --set always_yes yes --set changeps1 no
-              conda config --add channels omnia
-              conda config --add channels conda-forge
-            displayName: Add relavent Channels
+            displayName: Set conda config
 
           - bash: |
               conda update -c defaults conda
               conda update --all
-              conda create -n test-environment
-              source activate test-environment
-              conda install --yes python=$(python.version) --file requirements-dev.txt
-              conda install 'hoomd<3'
+              sed -i -E 's/  - python.*/  - python='$(python.version)'/' environment-dev.yml
```

Comment:
Can we make this regex a bit whitespace proof? Not that it doesn't work but if there's more than one space in the yml file it will not work. Having said that, since we have control over the `yml` file. Its not absolutely necessary to do so. For example, changing the above `regex` to 
``` sed -i -E 's/ -\s*?python.*$/ - python="$(python.version)"/' environment-dev.yml``` 
works for cases like this as well.
```yml
name: mbuild-dev
channels:
  - conda-forge
dependencies:
  - bump2version
  - codecov
  - ele
  - foyer
  - garnett>=0.7.1
  - gsd>=1.2
  - hoomd=2.9.4
  - intermol
  - mdtraj
  - networkx
  - nglview>=2.7
  - numpy
  - openbabel>=3.0.0
  - oset
  - packmol>=18
  - parmed>=3.4.0
  - pip
  - protobuf
  - py3Dmol
  - pycifrw
  - pytest
  - pytest-azurepipelines
  - pytest-cov
  -  python<=3.8
  - scipy

```

Reply:
good point!